python/tk_desktop/extensions/__init__.py

```diff
@@ -21,25 +21,7 @@
     # The PTR desktop app that comes with PySide2 does not require
     # any helper code to have the application move the foreground
     # properly.
-    try:
-        import PySide2  # noqa
-    except Exception:
-        # We're in a PySide 1 version of the PTR desktop app, so try
-        # to import the old osutils
-        try:
-            # This library links againts Qt, so the version number is
-            # suffixed to the module. If in the future we need to
-            # have an extension for qt5, we'll avoid a name clash.
-            from .linux_python2_qt4 import osutils
-
-            use_mocked_osutils = False
-        except Exception as e:
-            logger.warning("Could not load osutils: %s", e, exc_info=True)
-    else:
-        # PySide 2 build of desktop does not require help
-        # making the application go in the foreground/background, so we
-        # don't need to import an extension module.
-        pass
+    import PySide2  # noqa
```

Comment:
If I look at the rest of the file, this import does nothing. Similar to what we have on the `else` block (for Windows). 

I updated `osutils` last time for macOS and I can confirm this is the only OS when we make use of it. 

I suggest to only leave the `if sys.platform == "darwin":` block. Sorry about asking many changes on this.